### DISCUSSION TITLE
How to add charts to Quasar project  with chart.js ?

### DISCUSSION THREAD

**Discussion by CodeSengel** [quasarframework/quasar#11898] [General - Components / Directives / etc] [answered]
Hello, 

I would like to add charts to my quasar project. I tried to use chart.js, but I can't make it.
I've done a lot of research on the internet, but I haven't found anything interesting wich explains clearly how it works. 
so please, if you have a document or a link to a tutorial, please share it. 

thank you in advance, 
CodeSengel

**Comment by MilosPaunovic** [comment]
Hello, what exactly is troubling you? Have you tried to implement any solution yet?

**Reply by CodeSengel** [reply]
yes I tried several solutions, but as I am new to quasar and view it is not clear for which file should be modified and how to do it ... etc.
so, the question is sample : do you have any document or link to a tutorial  😄 

Thank you for your feedback 

CodeSengel

**Reply by metalsadman** [reply]
just look for already vue 3 compatible one and reduce the headache. cough* [vue-apexcharts](https://www.npmjs.com/package/vue3-apexcharts)

**Comment by CodeSengel** [comment, marked as answer]
Hello , 

Thank you for the answers.
The problem was that i was using vue 3 , and all the solutions I was trying to implement were only applicable on view 2 . 

you can see the example in this link : https://github.com/patrickmonteiro/quasar-apexcharts/tree/quasar-v2-vue3 

good luck
CodeSengel
